observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
+
+## Overview
+
+Dashboard provides "at glance" view of Key Performance Index(KPI) in accrodance with goals and objectives of business processes. Typically this includes metrices and key data points to monitor the health of a business, department or specific process. Dashboard usually connects to one or multiple data sources allowing the creation of visual representation of data. This is important in observability as a good dashboard will be able to 
+
+- show trends
+- identify patterns(user, usage, search etc)
+- measure efficiency easily
+- identify data outliers and correlations
+- view health state or performance of the system
+- give an outlook of the KPIs that is important to a business/process
+
+which otherwise is difficult to understand and consume relating back to business goal or KPI measure. Common questions to ask while building dashboard would be:
+- Where did my user spend most of their time at?
+- What is my user searching?
+- How do I better help my team with alerts and troubleshooting?
+- Is my system healthy for the past one day/week/month/quarter?
+
+## Best Practices
+
+1. Identify goals or KPI measurement. Identifying goals or KPI helps in defining what needs to be achieved. Here are some examples - remote call latency, average CPU/GPU consumption per time unit.
```

Comment:
1) Seems like the examples you are providing here are mostly system metrics (CPU, GPU, Latency, etc) and while they could perfectly be valid KPIs, KPIs are usually attached to business goals, for example: "orders processed", "messages sent", "successful searches", etc.
2) Nit: `KPIs` in plural (2 times in the paragraph).

Reply:
I looked into the definition & you're right. Addressed with other examples